lattice-control/lattice-control-interface.smithy

```diff
@@ -149,6 +152,79 @@ operation SetRegistryCredentials {
     input: RegistryCredentialMap
 }
 
+/// Instructs the provider to store the NATS credentials/URL for a given lattice. This is
+/// designed to allow a single capability provider (or multiple instances of the same) to manage
+/// multiple lattices, reducing overhead and making it easier to support secure multi-tenancy of
+/// lattices.
+operation SetLatticeCredentials {
+    input: SetLatticeCredentialsRequest
+    output: CtlOperationAck
+}
+
+/// A request to advertise/publish a link definition on a given lattice.
+structure AdvertiseLinkRequest {
+    /// The ID of the lattice for this request
+    @required
+    @serialization(name: "lattice_id")
```

Comment:
I believe all these serializations are unnecessary. They look like the default format the Rust codegen will use, and would be confusing for codegen in other languages (e.g. Go)

Reply:
:thinking:

Reply:
I have removed the aberrant, deviant, unnecessary serialization tags.